packages/server/src/adapters/aws-lambda/getPlanner.ts

```
@@ -1,11 +1,36 @@
+    }
+
+    type StreamifyHandler<TEvent = any, TResult = any> = (
+      event: TEvent,
+      responseStream: awslambda.HttpResponseStream,
+      context: Context,
+    ) => TResult | Promise<TResult>;
+
+    function streamifyResponse<TEvent = any, TResult = void>(
+      handler: StreamifyHandler<TEvent, TResult>,
+    ): StreamifyHandler<TEvent, TResult>;
+  }
```

ESLint

[error] ES2015 module syntax is preferred over namespaces.

(@typescript-eslint/no-namespace)